### DISCUSSION TITLE
Create a way to find which Cloud Vision annotations are highly correlated to manual annotations

### DISCUSSION THREAD

**Discussion by teolemon** [openfoodfacts/robotoff#1199] [Ideas]
### What

- [ ] Create a way to find which Cloud Vision annotations are highly correlated to manual annotations
- [ ] We can then review the candidates add them to our Cloud Vision label allowlists.

### Part of
- #599

**Comment by akashtadwai** [comment]
Hi @teolemon  I would like to work on this issue. Can you give more insight into it. I didn't understand how to check if they are related to manual annotations as described.
